### PR TITLE
Header, Intro Section and Skill Card

### PR DESCRIPTION
**Pull request by AbbyNyakara**
Created the Header section, The intro section and a skill card component for each of the skills (Grid Section). Also, How do I make the background color dynamic? As in, can I pass it as a prop?

### CODE REVIEW

File: src/components/Skills/Skills.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import img2 from '../../assets/pattern-illustrations.svg'
+import img3 from '../../assets/pattern-motion-graphics.svg'
+import img4 from '../../assets/pattern-photography.svg'
+import img5 from '../../assets/pattern-ui-ux.svg'
+import img6 from '../../assets/pattern-apps.svg'
+
+const Skills = () => {
+  return (
+    <div className="skills_grid">
+      <SkillCard image={img1} skill={"Graphic Design"}/>
```
Comment (gmagnenat) [inline src/components/Skills/Skills.jsx]: yes we can add the bgColor as a prop and setup utilities class for this.

File: src/components/SkillCard/SkillCard.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './SkillCard.scss'
+
+const SkillCard = ({image, skill}) => {
```
Comment (gmagnenat) [inline src/components/SkillCard/SkillCard.jsx]: we could get the bgColor prop here and use inline string concatenation with the skillCard class

...
```<div className={`skillCard bg-${bgColor}`}>```

File: src/components/Intro/Intro.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './Intro.scss'
+
+const Intro = () => {
+  return (
+    <div className='intro'>
+      <div className="intro__header">
```
Comment (gmagnenat) [inline src/components/Intro/Intro.jsx]: The wrapper div is not necessary and we can use a simple h1 for the text

File: src/components/Intro/Intro.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        <p>Design solutions made easy</p>
+      </div>
+      <div className="intro__text">
```
Comment (gmagnenat) [inline src/components/Intro/Intro.jsx]: I don't think the wrapper div is essential here either

File: src/components/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import logo from '../../assets/logo.svg';
+import './Header.scss'
+
+const Header = () => {
+  return (
+    <div className='header'>
+      <div className="header__logo">
+        <img src={logo} alt="" />
+      </div>
+      <div className="header__text">
```
Comment (gmagnenat) [inline src/components/Header/Header.jsx]: wrapper div not necessary

File: src/components/SkillCard/SkillCard.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
+@use '../../sass/abstracts/variables';
+
+.skillCard {
+  background-size: cover;
+  min-width: 300px;
+  min-height: 300px;
```
Comment (gmagnenat) [inline src/components/SkillCard/SkillCard.scss]: we will have to fix these sizes with grid instead..

File: src/components/SkillCard/SkillCard.scss [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.skillCard__image {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+}
+
+.skillCard__text {
+  position: absolute;
```
Comment (gmagnenat) [inline src/components/SkillCard/SkillCard.scss]: you can use flex-box and `margin-top: auto` for this.

**Review by gmagnenat — CHANGES_REQUESTED** [review]

File: src/components/SkillCard/SkillCard.scss (hunk 6 above)
Comment (AbbyNyakara) [reply src/components/SkillCard/SkillCard.scss:6]: removing the min-width and min-height displaces the text for the cards

Comment (AbbyNyakara) [reply src/components/SkillCard/SkillCard.scss]: Used flex-box as suggested to align the text and image inside the skill card

File: src/components/SkillCard/SkillCard.scss (hunk 7 above)
Comment (AbbyNyakara) [reply src/components/SkillCard/SkillCard.scss]: Interesting play with flex-box. Took me some reading to figure out..✅

File: src/components/Skills/Skills.jsx (hunk 1 above)
Comment (AbbyNyakara) [reply src/components/Skills/Skills.jsx]: Not familiar with utility classes. Cant figure out how they work

**Review by gmagnenat — APPROVED** [review]
I'll merge the current version and we'll continue on this.

File: src/components/SkillCard/SkillCard.jsx (hunk 2 above)
Comment (gmagnenat) [reply src/components/SkillCard/SkillCard.jsx]: I'll add them with comments.

**Pull request merged**